src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFolderProperty.java

```diff
@@ -65,21 +66,12 @@ private Set<String> getInheritedClouds() {
     @SuppressWarnings("unused") // Used by jelly
     @Restricted(DoNotUse.class) // Used by jelly
     public List<UsagePermission> getEffectivePermissions() {
```

Comment:
Looks unused now that you moved it to descriptor.

Reply:
Can you remove it as it is unused?

Reply:
Sorry, absolutely forgot about this no longer used method, removed